packages/sentry/sentry.js

```diff
@@ -1,11 +1,11 @@
 import * as Sentry from '@sentry/browser';
 
 export const boot = () => {
-  if (window.app.env === 'production') {
+  if (window.app.trackingSentry) {
     Sentry.init({
       dsn: window.app.trackingSentry,
       release: window.app.revision,
-      environment: window.app.env,
+      environment: `${window.app.env}-${window.app.location}`,
```

Comment:
This is to have more specific information about the environment within the same sentry project.